### ISSUE TITLE
Getting logger error when no logger setup

### ISSUE THREAD

**Issue by Kyah** [ssato/python-anyconfig#13]
Hi,
I'm starting a new project and i'll use anyconfig for my conf file handling but I have this message every time I start my app (which have no logging enabled yet) : " No handlers could be found for logger "anyconfig.api" "

Here is my only line refering to anyconfig : 
    settings.config = anyconfig.load([default_conf, user_home_conf, args_conf],
                                     "yaml", ignore_missing=True,
                                     merge=anyconfig.MS_DICTS)

Do I need to add something here or is there an error inside anyconfig?

Thanks

**Comment by ssato** [comment]
Thanks for your report!

I confirmed that same message was out w/ the following code:
```
ssato@localhost% cat /tmp/t.py
#! /usr/bin/python -tt
import anyconfig

c = anyconfig.load("/etc/") or {}
anyconfig.dump(c, "/dev/null", "yaml")
ssato@localhost% /tmp/t.py
No handlers could be found for logger "anyconfig.api"
ssato@localhost%
```

Although I'm not sure this is the same symptom you come across, I'll take some time to look into further. Please wait for a while.

**Comment by ssato** [comment]
NOTE: I think I found the cause. The choices looks to make log messages from anyconfig suppressed by default or to force users to configure logging; I prefer the former but maybe it brings some side effects along with that change. I'll take some time to check it further.

[https://docs.python.org/2/howto/logging.html#library-config]

**Comment by ssato** [comment]
Could you please try the git HEAD version in the master branch which I've committed related fixes
or the next 0.0.7 release will be released soon (I'm planning to release it this weekend)
if you can wait ?

**Comment by Kyah** [comment]
I've tested the new version available in the master branch and the message isn't there anymore. 

Thanks :+1:

**Comment by ssato** [comment]
Excuse me to late reply. Thanks for confirmation. I'll push that fix into the next release.

**Comment by ssato** [comment]
I've just released 0.0.7 contains the fix for this issue and I close this.
